### DISCUSSION TITLE
Report Abuse Malware

### DISCUSSION THREAD

**Discussion by Serialc0de** [community/community#126769] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Good evening, I am the same person who reported the profile "Mataraeosz (user)".
Well, this person persistently continues to forward Malware .rcs on the github platform, as I asked the Mataraeosz profile was deleted.
Now I would like to report the same person who continues to forcibly infect civilian users on your platform with the same Malware. Unfortunately I can't help but report it, but you can report it because it is spreading not only on github but also on other serious platforms.
This person created another profile after my report, the profile is this:

https://github.com/HarsTown

This client of yours relentlessly instructs and spreads persistent malware on the github platform, infecting innocent people and stealing confidential files and information.
I seriously ask that you take action. 

Thank you.

**Comment by LiteBrite82** [comment, marked as answer]
Hi @Serialc0de 

Thanks for looking out for the community and letting us know!

**Reply by LiteBrite82** [reply]
The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!

**Reply by Serialc0de** [reply]
I always report correctly following the community rules by facilitating my reports through the user report.
